css-layouts/layout-basics/src/exercises/website-challenge/solution/website-challenge-solution.css

```diff
@@ -0,0 +1,195 @@
+/*
```

Comment:
You probably don't need `Design_mock.png` and `Design_mock_with_annotations.png` since you converted those to use image hosts.

Reply:
Actually those I didn't because I figured it was easier to see them in the editor rather than visit imgur